app/src/components/Home/CoverPage.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import $ from 'jquery';
 import CoverPageStyle from '../../../styles/components/c-cover-page.scss';
-import Slider from 'react-slick';
+import Slider from '../../lib/react-slick.min';
```

Comment:
This is neccesary for stop the slider when we click dots.
That action is only possible with this version of react-slick because They not have release of this version... @tiagojsag